samples/bot-types-of-cards/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.14",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.bottypesofcards",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Types Of Cards",
+    "full": "Types Of Cards"
+  },
+  "description": {
+    "short": "Types Of Cards",
+    "full": "This sample shows the feature where user can send types of cards using bot."
```

Comment:
This sample shows the feature where user can send different types of cards using bot.

Reply:
fixed